.github/workflows/python_package.yml

```diff
@@ -3,11 +3,11 @@ name: Python-package
 on:
   push:
     branches:
-    - master
+      - master
```

Comment:
please revert all of these indentation and quoting changes, unless you can explain specifically how they're related to the goals of this PR